tests/rptest/services/redpanda.py

```diff
@@ -190,24 +206,7 @@ def start_node(self, node, override_cfg_params=None):
         if self.coproc_enabled():
             self.start_wasm_engine(node)
 
-        cmd = (f"nohup {self.find_binary('redpanda')}"
-               f" --redpanda-cfg {RedpandaService.CONFIG_FILE}"
-               f" --default-log-level {self._log_level}"
-               f" --logger-log-level=exception=debug:archival=debug:io=debug "
-               f" --kernel-page-cache=true "
-               f" --overprovisioned "
-               f" --smp {self._num_cores} "
-               f" --memory 6G "
-               f" --reserve-memory 0M "
-               f" >> {RedpandaService.STDOUT_STDERR_CAPTURE} 2>&1 &")
-
-        # set llvm_profile var for code coverage
-        # each node will create its own copy of the .profraw file
-        # since each node creates a redpanda broker.
-        if self.cov_enabled():
-            cmd = f"LLVM_PROFILE_FILE=\"{RedpandaService.COVERAGE_PROFRAW_CAPTURE}\" " + cmd
```

Comment:
was this intentional? @NyaliaLui can you review?

Reply:
`"LLVM_PROFILE_FILE=\"{RedpandaService.COVERAGE_PROFRAW_CAPTURE}\"` was intentional. Otherwise the data file is written to a seperate location. That block of code should only execute when coverage is enabled though.

Reply:
sorry, it is a merge skew